src/EnergyPlus/DataLoopNode.hh

```diff
@@ -615,6 +617,8 @@ namespace DataLoopNode {
         "SetpointManager:SingleZone:OneStageCooling",
         "SetpointManager:SingleZone:OneStageHeating",
         "SetpointManager:SingleZone:Reheat",
+        "SetpointManager:SystemNodeReset:Temperature",
```

Comment:
Please move these to the `.cc` file. Thx!

Reply:
Hi Amir, thanks for your feedback. This `DataLoopNode.hh` file doesn't have `.cc` file so we didn't move this to `.cc` file but the other one is moved to `.cc` file. I hope this works.